api/app/forecasts/noon_forecasts_summaries.py

```diff
@@ -38,12 +37,10 @@ def create_noon_forecast_summary(station: WeatherStation,
     for date in nested_dict:
         percentile_values = NoonForecastSummaryValues(
```

Comment:
could you please rename this variable, since it's no longer percentiles but min/max?

Reply:
Sure!